biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -240,4 +254,66 @@ private void init() {
 			throw Exceptions.duck(ioe);
 		}
 	}
+
+	public int getRelease() {
```

Comment:
There is already a method that gets the release from an EE automatically so no need to maintain this manually...

Reply:
that does not cover < 9

Reply:
Also please look at `test.model.EETest.checkEEHasValidRelease(EE)` that is even covered by a test...